Cargo.toml

```diff
@@ -8,9 +8,11 @@ name = "cross"
 repository = "https://github.com/cross-rs/cross"
 version = "0.2.1"
 edition = "2021"
+readme = "docs/crates_io.md"
```

Comment:
why?

Reply:
This is so when published on `crates.io`, it ensures that the README will display [this](https://github.com/Alexhuszagh/cross/blob/14857a333d6ad81e04b0af89aa9a3573bca07193/docs/crates_io.md). On `docs.rs`, it will look similar to the following image:

![Screenshot_20220608_091729](https://user-images.githubusercontent.com/9440903/172639769-8d3e2913-a33b-419a-ab89-a1332455352f.png)

Reply:
the warning would be nice to have in the documentation, but for crates.io it's fine to only have the regular readme I think.

It's not needed to specify README.md as the readme in the manifest, see https://github.com/cross-rs/cross/pull/563